ringo_printtemplate/views.py

```diff
@@ -126,9 +129,13 @@ def _render_template(request, template, item):
     """
     out = StringIO.StringIO()
     data = load_file(template.data)
-    temp = Template(StringIO.StringIO(data), out)
-    temp.render({"item": item, "print_item": PrintValueGetter(item, request)})
-    return out
+    mime = check_mime_type_from_buffer(data)
+    print(mime)
+    if mime == "application/vnd.oasis.opendocument.text":
+        temp = Template(StringIO.StringIO(data), out)
+        temp.render({"item": item, "print_item": PrintValueGetter(item, request)})
+        return out
+    return None
```

Comment:
I think this is a critical moment: We are about to let the converter unzip data on the server. So we should at least check at this moment that what we are about to do is roughly what we want.